mailpoet/assets/js/src/subscribers/subscribers.jsx

```diff
@@ -30,5 +30,6 @@ function App() {
 const container = document.getElementById('subscribers_container');
 
 if (container) {
+  registerTranslations();
```

Comment:
I'm wondering if there is some system solution for registering translations. Do you have any idea?
When I'm checking this commit, there is a lot of code repetition. At the same time, I'm thinking about what can happen when a developer forgets to add calling this function.

Reply:
basically, this is needed per page/application, and since we have a single application per each menu entry, this is needed, but we need to do it once per each menu entry, so while I see your concern, the probability of it happening often is very low.